### PR TITLE
Included default route template in docs

### PR DESCRIPTION
**Pull request by henkmollema**
Added the exact route being used in the
`BuilderExtensions.UseMvcWithDefaultRoute()` method.

#1885

### CODE REVIEW

**Comment by henkmollema** [comment]
@Eilon @kichalla 

As discussed in 53f3a91f01c740f3cbf5209a9eeeefd614a6cbc2

**Comment by Eilon** [comment]
:shipit: 

@kichalla can you merge?

File: src/Microsoft.AspNet.Mvc/BuilderExtensions.cs [hunk 1]
```diff
@@ -31,7 +31,7 @@ public static IApplicationBuilder UseMvc([NotNull] this IApplicationBuilder app)
 
         /// <summary>
         /// Adds MVC to the <see cref="IApplicationBuilder"/> request execution pipeline
-        /// with a default route.
+        /// with a default route using the following template: '{controller=Home}/{action=Index}/{id?}'.
```
Comment (kichalla) [inline src/Microsoft.AspNet.Mvc/BuilderExtensions.cs]: Should we include the route name too as it can be used in link generation?

Comment (henkmollema) [inline src/Microsoft.AspNet.Mvc/BuilderExtensions.cs]: Was thinking about that. Perhaps the most explicit option.

Comment (Eilon) [inline src/Microsoft.AspNet.Mvc/BuilderExtensions.cs]: Sure. "... with a default route named 'Xyz' and the following template: ..."

**Comment by henkmollema** [comment]
@kichalla @Eilon updated and rebased.

**Comment by kichalla** [comment]
:shipit:

**Pull request merged**